content/en/mpas/getting_started.md

```diff
@@ -0,0 +1,394 @@
+The certificate should be named `ocm-registry-tls-certs` and should be placed in the `mpas-system`
+and `ocm-system` namespaces. You can use [syncing-secrets-across-namespaces](https://cert-manager.io/docs/tutorials/syncing-secrets-across-namespaces/) guide to sync the certificate between namespaces.
+
+
+### Deploy podinfo application
+
+The [podinfo application](https://github.com/stefanprodan/podinfo) has been packaged
+as an ocm component and can be retrieved from [Github](ghcr.io/open-component-model/podinfo).
+
+<!--TODO: Surface the SA from the podinfo app into values.yaml, in order to use imagePullSecrets
+generated by a Target.
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  enabled: false
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+  # List of image pull secrets if pulling from private registries
+  imagePullSecrets: []
+-->
+
+<!--TODO: Add project creation command
+
+```bash
+mpas create project podinfo-project --export > podinfo-project.yaml
+```
+We should also have a command to create a component subscription as well as a product deployment generator.
+--->
+
+1. Create a secret containing your Github credentials that will be used by MPAS to
+create your project repository.
+
+```bash
+kubectl create secret generic \
+  github-access \
+  --from-literal=username=$GITHUB_USER \
+  --from-literal=password=$GITHUB_TOKEN \
+  -n mpas-system
+```
+
+2. Create a project that will contain the podinfo application.
+
+Let's create a directory for the project:
+
+```bash
+mkdir -p ./clusters/my-cluster/podinfo
+````
+
+Then, create a `project.yaml` file in the `./clusters/my-cluster/podinfo` directory:
+
+```bash
+cat <<EOF >> ./clusters/my-cluster/podinfo/project.yaml
+apiVersion: mpas.ocm.software/v1alpha1
+kind: Project
+metadata:
+  name: podinfo-application
+  namespace: mpas-system
+spec:
+  flux:
+    interval: 1h
+  git:
+    provider: github
+    owner: $GITHUB_USER
+    isOrganization: false
+    visibility: public
+    maintainers:
+    - $GITHUB_USER
+    existingRepositoryPolicy: adopt
+    defaultBranch: main
+    credentials:
+      secretRef:
+        name: github-access
+    commitTemplate:
+      email: mpas@mpas.ocm.software
+      message: Initializing Project repository
+      name: mpas-admin
+  prune: true
+EOF
+```
+
+Then, apply the project to the clustern in a gitOps fashion:
+
+```bash
+git add --all && git commit -m "Add podinfo project" && git push
+```
+
+`Flux` will detect the changes and apply the project to the cluster.
+
+This will create in the cluster a `namespace` for the project, a `service account`, and RBAC.
+It will also create a GitHub repository for the project, and configure `Flux` to manage the project's resources.
+
+3. Add the needed secrets to the namespace
+
+`Flux` is used to deploy all workloads in a gitOps way. In order for flux to access the internal
```

Comment:
You're using `Flux` and flux and Flux interchangeably. I think it might be better if you stick to a format?